engine/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -314,10 +314,10 @@ assumes that the Docker daemon is in the `stopped` state.
 
 14. Configure the Docker daemon with specific devicemapper options.
 
-    There are two ways to do this. You can set options on the command line if you start the daemon there:
+    Now that your storage has been configured, we have to instruct the Docker daemon to use it.  There are two ways to do this. You can set options on the command line if you start the daemon there:
```

Comment:
Please avoid the use of second person ("we"). I suggest "Now that your storage is configured, configure the Docker daemon to use it."

Nit: Please use only a single space between sentences.